app/main.py

```diff
@@ -1 +1,55 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+
+    def __init__(self,
+                 name: str,
+                 is_hungry: bool = True
+                 ) -> None:
+        super().__init__(name, appetite=3)
+        self.is_hungry = is_hungry
```

Comment:
You must pass this parameter into super

Reply:
What do you mean? Which parameter? If you mean "is_hungry" then it doesn't work this way. Could you point more precisely please?